client/src/sass/overrides.sass

```diff
@@ -43,15 +43,15 @@
   padding: 0 10px 0 10px !important
 
 .v-btn
-  color: white !important
-  background-color: $link !important
-  height: 40px !important
-  padding: inherit 28px inherit 28px !important
-  border-bottom-width: 2px !important
+  min-width: 0
+  text-transform: none
+  font-weight: 400
+  letter-spacing: normal
+  font-size: .875rem
 
-.v-btn.normal
-  color: white !important
-  background-color: $grey-5 !important
+.v-icon + span,
```

Comment:
This is quite a change. Do you have a list of buttons affected (for our review and also for QA to verify)?

Reply:
Yea, i'm pretty it affects almost ALL the buttons but we have a few styling ticket changes to come. This was just the first bit to at least bring the btn's in sync with Business/Filings etc, as it significantly affected the appearance of the common components.

Imo i think its ok that a few btn's look a bit different until we dig in to those styling tickets but totally your call.

Reply:
I'm OK with this change, but it would be nice to verify WHAT changed so we can look at it closely (for example, what if the new style conflicts with a local style and the button is all messed up now?).

Reply:
Aw nuts. I see there are 38 instances of a v-btn.

Do you think we should just risk this change and fix things later?

Reply:
hmm, on second thought i think i'll try and isolate the header. 
We will still fix this later but in the meantime, best to leave whats existing untouched considering the scheduling etc.

Reply:
You're 100% right on the local conflicts. I imagine that would be extensive.

Reply:
Restored original btn styles and isolated the Header btns, in next commit 👍